### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by fbonzon**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,7 +329,7 @@ Provider  | Name | IPv4? | IPv6? | Multiple? | Terms | Notes
 [FreeGeoIp](http://freegeoip.net/) | `free_geo_ip` | yes | yes | no
 [GeoIPs](http://www.geoips.com/en/) | `geo_ips` | yes | no | no | requires API key
 [GeoIP2](https://www.maxmind.com/en/geoip2-databases) (Maxmind) | `maxmind_geoip2` | yes | yes | no
-[GeoPlugin](http://www.geoplugin.com/) | `geo_plugin` | yes | yes | no
```
Comment (fbonzon) [inline README.md]: Changing a non-breaking space to regular space

**Comment by toin0u** [comment]
:+1:

**Comment by Baachi** [comment]
:+1:

**Comment by willdurand** [comment]
thanks!

**Pull request merged**